samples/tab-user-profile/ts/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-user-profile-ts",
+    "source": "officeDev",
+    "title": "Starter solution: Personal tab in TypeScript",
+    "shortDescription": "Create React App personal tab in TypeScript",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-user-profile/ts",
+    "longDescription": [
+      "Create React App personal tab in TypeScript"
```

Comment:
This sample app shows how to open a user profile information card at a specified position in Outlook using personal tab app.

Reply:
Fixed